docs/inputs/input-group.md

```diff
@@ -42,3 +42,25 @@ As an example, the image below shows an input group with 5 different inputs (two
 ![img alt](/docs/inputs/input_group.png)
 
 </div>
+
+### Conditional Input Group
+
+| :trophy: This is a [premium](https://mecsimcalc.com/premium) features |
+| --------------------------------------------------------------------- |
+
+Click the pencil icon near the top right of an input group and the **Edit Input Group** popup will appear.  
```

Comment:
It might be easier to replace "pencil icon" with an image of the icon embedded inline